### DISCUSSION TITLE
koreader in Linux - Switching off  display of keyboard navigation shortcuts

### DISCUSSION THREAD

**Discussion by PrasiNV** [koreader/koreader#12155] [Q&A] [answered]
Hi,
I have recently started using koreader in Linux (appimage). 
File browser in mosaic mode displays the keyboard navigation shortcuts (Q, W, E, R, T, Y,......). As I use the app with a mouse most of the time, I wonder if there is a way to switch off the keyboard navigation shortcuts from being displayed?

**Comment by benoit-pierre** [comment]
Unfortunately, AFAIK, that's not configurable.

**Reply by mergen3107** [reply]
Yes, it is possible! There is a user patch for that. Let me find it

**Reply by mergen3107** [reply, marked as answer]
Here: https://github.com/koreader/koreader/issues/10177

**Reply by mergen3107** [reply]
The way I found this:
1) Go to KOReader User Guide: https://koreader.rocks/user_guide/
2) Here find User patches: 
https://koreader.rocks/user_guide/#userpatches
3) Go to User patches link on GitHub:
https://github.com/koreader/koreader/issues?q=+label%3A%22User+patch+available%22
4) Search here. There are a lot of interesting patches :D

**Reply by PrasiNV** [reply]
Thank you very much. I was able to apply this patch and it worked.

**Comment by hius07** [comment]
I'd better patch it here, to keep the keyboard enabled in other places.
https://github.com/koreader/koreader/blob/75d3e4c2dcc184281fff490431e98123b6ba0c26/frontend/ui/widget/menu.lua#L599

**Reply by PrasiNV** [reply]
Thank you for the reply. 

I am using koreader (appimage) in Linux. I am unable to find the menu.lua file in the following location ~/.config/koreader

**Reply by hius07** [reply]
It should be a userpatch, do not modify the source code.
